test/ref-list.json

```diff
@@ -9080,6 +9080,8 @@
     "INSPIRE-MD-20131029",
     "inspire-md-20170302",
     "INSPIRE-MD-20170302",
+    "interledger-protocol",
```

Comment:
Fine to keep those in, but for future reference, we have a bot that's going to add them in automatically.

Reply:
👍  just wanted to make sure the tests were still passing as expected